core-services/prow/02_config/_plugins.yaml

```diff
@@ -2,6 +2,7 @@ blunderbuss:
   max_request_count: 2
   request_count: 2
   use_status_availability: true
+branch_cleaner: {}
```

Comment:
this is why you need extra approvals @Tal-or 
Do we really need this change?

Reply:
It was automatically generated

Reply:
I'll try remove it, let's if it'll pass